polkadot/runtime/westend/src/lib.rs

```diff
@@ -2408,6 +2410,4 @@ mod clean_state_migration {
 				"State migration clean.",
 			);
 			Ok(())
-		}
-	}
-}
```

Comment:
?

Reply:
I have no idea what happened there, I ran `cargo +nightly fmt --all` but those lines were still there for me locally. Fixed now though.